src/types/rest/response/private-block-trading.ts

```diff
@@ -97,3 +97,43 @@ export interface GetBlockQuoteResult {
   clRfqId: string;
   traderCode: string;
 }
+
+export interface BlockMakerInstrumentData {
+  uly?: string;
+  instId?: string;
+  maxBlockSz?: string;
+  makerPxBand?: string;
+}
+
+export interface BlockMakerInstrumentSettings {
+  instType: string;
+  includeALL: boolean;
+  data: BlockMakerInstrumentData[];
+}
+
+export interface SetMmpConfigResponse {
+  timeInterval: string;
+  frozenInterval: string;
+  countLimit: string;
+}
+
+export interface GetMmpConfigResponse {
+  frozenInterval: string;
+  mmpFrozen: boolean;
+  mmpFrozenUntil: string;
+  countLimit: string;
+  timeInterval: string;
+}
```

Comment:
```suggestion
export interface MmpConfig {
  frozenInterval: string;
  mmpFrozen: boolean;
  mmpFrozenUntil: string;
  countLimit: string;
  timeInterval: string;
}
```

if the response is the "thing", call it what it is